packages/calcite-components/src/components/input/input.tsx

```diff
@@ -128,8 +128,7 @@ export class Input
    *
    * When not set, the component will be associated with its ancestor form element, if any.
    */
-  @Prop({ reflect: true })
-  form: string;
+  @Prop({ reflect: true }) form: string;
```

Comment:
@jcfranco do you know why this keeps ending up on a new line? Lol

Reply:
It looks like it's the result of my habit of placing decorators above of their target and https://github.com/stencil-community/stencil-eslint/blob/main/docs/decorators-style.md not auto fixing for some reason. Does it auto fix this for you when explicitly running the linter?